source/common/http/conn_manager_impl.cc

```diff
@@ -869,6 +869,13 @@ absl::optional<uint64_t> ConnectionManagerImpl::HttpStreamIdProviderImpl::toInte
       *parent_.request_headers_);
 }
 
+absl::optional<uint32_t> ConnectionManagerImpl::HttpStreamIdProviderImpl::getCodecStreamId() const {
+  if (parent_.response_encoder_ == nullptr) {
+    return absl::nullopt;
+  }
+  return parent_.response_encoder_->getStream().codecStreamId();
```

Comment:
Does this not return the codec stream id for H/3 too?

Reply:
I was just referring to this comment https://github.com/envoyproxy/envoy/pull/42704/files#diff-8acb82193b72884d79bc2644e78076864065d66451039980fbadd7180b371e2dR473 which suggests H/3 isn't supported yet.  Should we update the comment?